site/src/components/pages/dashboard/top-navigation-tabs.tsx

```diff
@@ -43,6 +42,7 @@ export const TopNavigationTabs: FunctionComponent = () => {
               component="a"
               onClick={(event: MouseEvent) => {
                 event.preventDefault();
+                void router.push(tabHref);
```

Comment:
So it always opens the tab's href, even if tab is already active. 
Which means, when user is on `/blocks/publish`, clicking the `Blocks` tab will take the user back to the tabHref, which is `/blocks`